git/client_test.go

```diff
@@ -471,22 +471,22 @@ func TestClientCommits(t *testing.T) {
 		{
 			name:        "get commits",
 			cmdStdout:   "6a6872b918c601a0e730710ad8473938a7516d30,testing testability test",
-			wantCmdArgs: `path/to/git -c log.ShowSignature=false log --pretty=format:%H,%s --cherry SHA1...SHA2`,
+			wantCmdArgs: `path/to/git -c log.ShowSignature=false log --pretty=format:%H,%s,%b --cherry SHA1...SHA2`,
 			wantCommits: []*Commit{{
 				Sha:   "6a6872b918c601a0e730710ad8473938a7516d30",
 				Title: "testing testability test",
 			}},
```

Comment:
This is not affected also if body is missing! I added a new test to check body is read correctly :)

Reply:
would adding `--no-merges` to the log command help improve the message generated ?

Reply:
I like it, but not sure if this is expected behaviour by default

Reply:
The Web UI releases page - Generate feature does not include merge commits so i don't think it would be completely unexpected as such and merge commits messages are not very useful to begin with